### DISCUSSION TITLE
Style applied after page loads?

### DISCUSSION THREAD

**Discussion by ewolfe1** [thegetty/quire#947] [Get Help] [answered]
A persistent problem that I've been seeing is that the styling is way off on each page load. It seems like maybe the elements are loading before the css/styling. It's not awful (1 second or less), but when it happens every time on every page, it's distracting and comes across as bad design. See attached for an example screenshot. 

Is this a known issue? Is it possibly something in my installation that I could address?

Thanks!

Macbook Pro, Sonoma 14.5
Quire = 1.0.0-rc.15
node = v20.16.0

![Screenshot 2024-08-23 at 9 18 29 AM](https://github.com/user-attachments/assets/41cf3f9f-9ad1-426d-a1ce-ae660e0259e8)

**Comment by geealbers** [comment, marked as answer]
Hi @ewolfe1, I'm happy to say that this is only an issue with the previewing process. Once you deploy your built site files to a web server you won't see this delay in the styles loading. 

If you want to test it out deploying your site and don't already have ready access to a web server, there are [instructions in the Quire docs](https://quire.getty.edu/docs-v1/site-deploy/#netlify) on doing pretty quick and easy manual deploys with Netlify Drop that you might want to check out.

**Reply by ewolfe1** [reply]
Excellent to hear! Thanks so much for the quick response and reassurance!
